de.wolfvollprecht.UberWriter.json

```diff
@@ -0,0 +1,135 @@
+{
+    "app-id": "de.wolfvollprecht.UberWriter",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.26",
+    "sdk": "org.gnome.Sdk",
+    "command": "uberwriter",
+    "rename-icon": "uberwriter",
+    "finish-args": [
+        "--socket=x11",
+        "--share=ipc",
+        "--filesystem=host",
+        "--env=IN_FLATPAK=1",
+        "--filesystem=xdg-run/dconf",
+        "--filesystem=~/.config/dconf:ro",
+        "--talk-name=ca.desrt.dconf",
+        "--env=DCONF_USER_CONFIG_DIR=.config/dconf",
+        "--env=PATH=/app/extensions/TexLive/bin:/app/extensions/TexLive/2018/bin/x86_64-linux:/app/bin:/usr/bin"
+    ],
+    "build-options" : {
+      "env": {
+        "PYTHON": "python3",
+        "IN_FLATPAK": "1"
+        }
+    },
+    "add-extensions": {
+        "de.wolfvollprecht.UberWriter.Plugin": {
+          "directory": "extensions",
+          "version": "stable",
+          "subdirectories": true,
+          "no-autodownload": true,
+          "autodelete": true
+    }
+    },
+    "modules": [
+        {
+        "name": "uberwriter",
+        "sources": [
+                {
+                    "type" : "git",
+                    "url" : "git://github.com/wolfv/uberwriter",
+                    "tag" : "v2.0",
+                    "commit": "23bde86ae1f344f1bcad30611c44efcd9d200411"
+                }
+            ],
+            "build-commands": [
+                "install -Dm644 flatpak/de.wolfvollprecht.UberWriter.appdata.xml /app/share/appdata/de.wolfvollprecht.UberWriter.appdata.xml "
+            ],
+            "post-install": [
+            "glib-compile-schemas /app/share/glib-2.0/schemas",
+            "install -d /app/extensions"
+            ]
+        },
+        {
+        "name": "pandoc",
+        "only-arches": [
+          "x86_64"
+            ],
+        "buildsystem": "simple",
+        "build-commands": [
+          "cp bin/pandoc /app/bin/pandoc",
```

Comment:
You are still just copying binaries...

Reply:
I thought we've determined that that's ok for now? Otherwise, we'd just take the binary for the Haskell compiler GHC and download that, in order to compile pandoc. It doesn't seem much more reproducible to me, for now. But once we get this shipped, I think it would be cool to use the script up there, by @lettier.